ast.c

```diff
@@ -321,8 +327,8 @@ static const ast_flag_info flag_info[] = {
 	{ ZEND_AST_ARROW_FUNC, 1, func_flags },
 	{ ZEND_AST_PROP_DECL, 1, modifier_flags },
 	{ ZEND_AST_PROP_GROUP, 1, modifier_flags },
-	{ ZEND_AST_CLASS_CONST_DECL, 1, visibility_flags },
-	{ ZEND_AST_CLASS_CONST_GROUP, 1, visibility_flags },
+	{ ZEND_AST_CLASS_CONST_DECL, 1, class_const_flags },
+	{ ZEND_AST_CLASS_CONST_GROUP, 1, class_const_flags },
```

Comment:
I realize this is pre-existing, but why does CONST_DECL have class_const_flags? Aren't they always on the GROUP?

Reply:
`AST_CLASS_CONST_GROUP     class, attributes      // version 80+`

CLASS_CONST_GROUP was added in version 80 to support attributes, because CONST_DECL->children is a list of nodes - earlier versions have no GROUP at all